core/src/main/java/io/questdb/log/LogAlertSocket.java

```diff
@@ -304,6 +308,69 @@ int getDefaultAlertPort() {
         return defaultPort;
     }
 
+
+    private static boolean isContentLength(CharSequence message, int lo, int hi) {
+        int len = hi - lo;
+        if (len < CONTENT_LENGTH.length) {
+            return false;
+        }
+        for (int i = 0; i < CONTENT_LENGTH.length; i++) {
+            if ((message.charAt(lo + i) | 32) != CONTENT_LENGTH[i]) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    @VisibleForTesting
```

Comment:
there is already available IntelliJ annotation `@TestOnly`, we could perhaps standardise on that? We already use their `@Nullable` etc, may as well use the one above